WorkspaceServer.Tests/NuGetPackageExtensionLoaderTest.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Threading.Tasks;
+using WorkspaceServer.Packaging;
+using Xunit;
+using System.IO;
+using XPlot.DotNet.Interactive.KernelExtensions;
+using WorkspaceServer.Tests.Kernel;
+using Xunit.Abstractions;
+using MLS.Agent.Tools;
+using Microsoft.DotNet.Interactive;
+using FluentAssertions;
+using Microsoft.DotNet.Interactive.Events;
+using System.Linq;
+
+namespace WorkspaceServer.Tests
+{
+    public class NuGetPackageExtensionLoaderTest : CSharpKernelTestBase
+    {
+        public NuGetPackageExtensionLoaderTest(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task Given_a_directory_loads_the_extensions_in_extensions_folder()
```

Comment:
I think the name `It_loads_the_extensions_in_extensions_folder` is simpler. The "given" wording tends to signal a BDD-style test, which this is not.